rqd/rqd/rqcore.py

```diff
@@ -1039,6 +1039,20 @@ def runDocker(self):
             command.replace(tempPassword, "[password]").replace(";", "\n"),
             prependTimestamp=rqd.rqconstants.RQD_PREPEND_TIMESTAMP)
 
+        # Handle memory limits. Cuebot users KB docker uses Bytes.
+        # Docker min requirement is 6MB, if request is bellow limit, give the frame a reasonable
```

Comment:
Change from:

`# Docker min requirement is 6MB, if request is bellow limit, give the frame a reasonable`

to:

`# Docker minimum requirement is 6MB, if the request is below the limit, give the frame a reasonable`